scheduler/cloudformation/template.go

```diff
@@ -235,6 +235,13 @@ func (t *EmpireTemplate) addTaskDefinition(tmpl *troposphere.Template, app *sche
 	cd := t.ContainerDefinition(app, p)
 	containerDefinition := cloudformationContainerDefinition(cd)
 
+	// If provided in the app environment, this role will be used when
+	// running tasks.
+	var taskRole interface{}
+	if arn, ok := app.Env["TASK_ROLE_ARN"]; ok {
```

Comment:
Yeah, maybe we should do that here, and then for those old ones honor the old way and the new prefixed way for a release before deprecating the old one or something.  To be fair, I think both of the 'old' cases are still pretty new, right?